shared/elements/gaia_grid/js/grid_dragdrop.js

```diff
@@ -666,6 +672,12 @@
               return;
             }
 
+            if (this.gridView._launchingApp) {
```

Comment:
Is it possible to set some flag on touchend and rely on that instead? The current debounce mechanism we have is a giant hack and should probably go away. Would prefer to avoid hooking into it if possible.